bottlecap/src/logs/flusher.rs

```diff
@@ -59,14 +63,18 @@ impl Flusher {
 
     async fn send(client: reqwest::Client, api_key: String, fqdn: String, data: Vec<u8>) {
         let url = format!("{fqdn}/api/v2/logs");
+        let mut encoder = Encoder::new(Vec::new(), 0).unwrap();
```

Comment:
This would create an encoder every time we are sending data, is it possible to reuse it somehow?

Reply:
I believe the encoder is finalized to get the compressed bytes and can't be reused